app/frontend/src/cljs/teet/contract/contract_common.cljs

```diff
@@ -12,12 +12,16 @@
             [teet.contract.contract-status :as contract-status]
             [teet.ui.format :as format]
             [teet.util.euro :as euro]
+            [clojure.edn :as edn]
             [teet.user.user-model :as user-model]))
 
 (defn contract-procurement-link
   [{:thk.contract/keys [procurement-number]}]
-  [common/external-contract-link {:href (str (environment/config-value :contract :state-procurement-url) procurement-number)}
-   (str (tr [:contracts :state-procurement-link]) " " procurement-number)])
+  (if (js/Number.isInteger (edn/read-string (subs procurement-number 0 1)))
```

Comment:
Is the edn dependency necessary? Would just `js/parseInt` and then cljs core `integer?` be enough?